### PR TITLE
feat: add server and channel schema

### LINKED ISSUE #15: [FEATURE] <add server and channel schema for twtcord>
### ⭐ Suggestion

Implementing server and channel details for twtcord and saving it to azure.

### 💻 Use Cases

Azure CosmosDB for automatic management.
Server side rendering using Nodejs.

### ❌ Related Problems

_No response_

### PR DESCRIPTION
**Pull request by AayushShuk1a**
Added Server and Channel Schema with following details:-

Server Details
-Server Name
-Server Profile
-Server Members
-Server TextChannels
-Server Owners
-Server VoiceChannels

Channel Details
-channelName
-newMessage
ServerName

Closes #15

### CODE REVIEW

File: server/Routes/Routes.js [hunk 1]
```diff
@@ -23,36 +24,41 @@ Router.post("/check", async (req, res) => {
...
+    console.log(err);
+  }
 });
+
+//Feeds Routes
 Router.get("/feeds", getFeeds);
 Router.post("/feeds", createFeeds);
 
+//Server Routes
+Router.get("/createserver", createServer);
```
Comment (chandankumar1307) [inline server/Routes/Routes.js:59]: Created Post request but using get. Fix this

**Review by chandankumar1307 — CHANGES_REQUESTED** [review]
Get it done ASAP

**Review by chandankumar1307 — APPROVED** [review]
Looks Good

**Review by SuperAayush — APPROVED** [review]
LGTM

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Review by bhavyastar — APPROVED** [review]
LGTM

**Review by chirag0002 — APPROVED** [review]
LGTM

**Pull request merged**